website/app/controllers/show.js

```diff
@@ -202,6 +204,20 @@ export default class ShowController extends Controller {
     // console.log('show didInsert', this.sections, this.tabs, this.tocs);
   };
 
+  applyCodeHighlighting() {
+    const allSections = document.querySelectorAll(
```

Comment:
suggestion: since what we're highlighting are code blocks (or code snippets, to be more precise) and since the word "section" is often used to indicate different parts of the pages (and is also an HTML tag) maybe we could call these entities `codeBlocks` or `codeSnippets` and a single item would be `codeBlock` or `codeSnippet`. What do you think?